lib/oppen/mixins.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Oppen
+  # Mixins.
+  module Mixins
+    # Rotates circular array and triples its size.
+    # This method is not for public use.
+    #
+    # @param arr [Array]
+    # @param offset [Integer] Rotation amount
+    #
+    # @return [Array(Array, Integer, Integer)] upsized array, lhs, rhs
```

Comment:
You checked the documentation output? Are we sure it's not supposed to be like this?

```suggestion
    # @return [Array<Array, Integer, Integer>] upsized array, lhs, rhs
```